megazords/ios/MozillaAppServicesTests/LoginsTests.swift

```diff
@@ -6,186 +6,5 @@ import XCTest
-            timesUsed: 0,
-            timeCreated: 0,
-            timeLastUsed: 0,
-            timePasswordChanged: 0
-        ))
-
-        let record1 = try! storage.get(id: id1)!
-
-        XCTAssertNil(record1.formSubmitUrl)
-        XCTAssertEqual(record1.httpRealm, "Something Something")
-    }
-
-    func testLoginEnsureValid() {
-        try! storage.unlockWithKeyAndSalt(key: encryptionKey, salt: salt)
-
-        _ = try! storage.add(login: Login(
-            id: "",
-            hostname: "https://www.example5.com",
-            password: "hunter5",
-            username: "cooluser55",
-            httpRealm: nil,
-            formSubmitUrl: "https://www.example5.com",
-            usernameField: "users_name",
-            passwordField: "users_password",
-            timesUsed: 0,
-            timeCreated: 0,
-            timeLastUsed: 0,
-            timePasswordChanged: 0
-        ))
-
-        let dupeLogin = Login(
-            id: "",
-            hostname: "https://www.example5.com",
-            password: "hunter3",
-            username: "cooluser55",
-            httpRealm: nil,
-            formSubmitUrl: "https://www.example5.com",
-            usernameField: "users_name",
-            passwordField: "users_password",
-            timesUsed: 0,
-            timeCreated: 0,
-            timeLastUsed: 0,
-            timePasswordChanged: 0
-        )
-
-        let nullValueLogin = Login(
-            id: "",
-            hostname: "https://www.example6.com",
-            password: "hunter3",
-            username: "\0cooluser56",
-            httpRealm: nil,
-            formSubmitUrl: "https://www.example6.com",
-            usernameField: "users_name",
-            passwordField: "users_password",
-            timesUsed: 0,
-            timeCreated: 0,
-            timeLastUsed: 0,
-            timePasswordChanged: 0
-        )
-
-        XCTAssertThrowsError(try storage.ensureValid(login: dupeLogin))
-        XCTAssertThrowsError(try storage.ensureValid(login: nullValueLogin))
-    }
-
-    func addLogin() -> String {
-        let login = Login(
-            id: "",
-            hostname: "https://www.example5.com",
-            password: "hunter3",
-            username: "cooluser55",
-            httpRealm: nil,
-            formSubmitUrl: "https://www.example5.com",
-            usernameField: "users_name",
-            passwordField: "users_password",
-            timesUsed: 0,
-            timeCreated: 0,
-            timeLastUsed: 0,
-            timePasswordChanged: 0
-        )
-        return try! storage.add(login: login)
-    }
-
-    func testListLogins() {
-        try! storage.unlockWithKeyAndSalt(key: encryptionKey, salt: salt)
-
-        let listResult1 = try! storage.list()
-        XCTAssertEqual(listResult1.count, 0)
-
-        _ = addLogin()
-
-        let listResult2 = try! storage.list()
-        XCTAssertEqual(listResult2.count, 1)
-    }
+    // TODO: Add migration tests here
```

Comment:
I won't block on this, but do we have a task we can add in here to ensure we don't forget to get this in?

Reply:
You should, lol! This was actually a note to myself. I'm going to add the tests before merging.